articles/machine-learning-and-deep-learning/index.md

```diff
@@ -0,0 +1,55 @@
+### Introduction
+
+Technology is growing at a fast pace in recent years. Due to this, it is now easy to get lost in the rumpus of technological terms that keep popping up every day. Machine learning and deep learning are among the fast-growing technologies. Besides this two, [Internet of Things (IoT](https://internetofthingsagenda.techtarget.com/definition/Internet-of-Things-IoT)), [blockchain](https://en.wikipedia.org/wiki/Blockchain), and [Artificial Intelligence](https://www.investopedia.com/terms/a/artificial-intelligence-ai.asp) (AI) are among the fast-growing technologies.
+
+New ideas in the tech world have led to various innovations that have sparked the nature of things. Machine learning and deep learning have attracted great attention over the years. The popularity of these two has led to various applications implementing its tools.
+
+This article will focus on understanding what machine learning and deep learning are. Both applications, differences, and functionality will be emphasized in this article.
+
+### Machine Learning and Artificial Intelligence
+
+Machine learning is a technological tool that uses algorithmic interpretations. This has helped in obtaining and using data to make and interpret final decisions. Machine learning processes are also dependent on the subsets of artificial intelligence. Several [synthetic intelligence](https://www.wikiwand.com/en/Synthetic_intelligence) aspects and modern [virtual assistants](https://www.investopedia.com/terms/v/virtual-assistant.asp) are created to achieve this.
+
+Machine learning uses Artificial Intelligence (AI) techniques that assist in decision making. Conventional algorithms in AI aid in performing mathematical computation and predictions. Machine learning depends on fed structured data. Conventional algorithms aids in achieving data mining and predictive analytics in a search.
+
+Unlike machine learning, AI can use [unstructured data](https://www.datamation.com/big-data/structured-vs-unstructured-data.html) and still be able to function. Deep learning does mimic the human brain's workings. This is because it uses this data to perform the processing of data. Machine learning aids in the detection of objects, language translation, and decision making. Machine learning is also able to recognize speech and make prompt decisions. The AI makes this possible this allows in no aspect of human supervision.
```

Comment:
Deep learning is a subset of machine learning which is based on deep neural networks. The basic unit of a neural network is a neuron. The neuron is biologically inspired by the structure of the neuron in the brain. It would be incorrect to say deep learning mimic's the brain.

Reply:
I changed the review on mention 'does mimic the brain'.